README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Introduction
 
-torchao is a library to create and integrate high-performance custom data types, layouts and kernels into their PyTorch workflows with up to **2x speedups** with **65%** less VRAM for [inference](#inference) and support for [training](#training)
+torchao is a library to create and integrate high-performance custom data types, layouts and kernels into your PyTorch workflows with up to **2x speedups** with **65% less VRAM** for [inference](#inference) and support for [training](#training)
```

Comment:
should we also have an index like thing at the top like tune does? https://github.com/pytorch/torchtune?tab=readme-ov-file#torchtune

Reply:
Done